packages/frontend/src/server/features/scaling/get-common-scaling-entry.ts

```diff
@@ -102,11 +166,11 @@ export function getCommonScalingEntry({
   }
 }
 
-function getStageOrder(stage: StageConfig | undefined): number {
-  if (stage?.stage === 'Stage 2' || stage?.stage === 'Stage 1') {
+function getStageOrder(stage: string | undefined): number {
```

Comment:
can't we have it as StageConfig? I would prefer to have the checks below typesafe

Reply:
I'm going to make another PR where I make some of those small things cleaner